### PR TITLE
Feat/#779 reg sample sidebar icons

### PR DESCRIPTION
**Pull request by ciaranschutte**
**Description of changes**

**Type of Change**

- [ ] Bug
- [ ] Styling
- [x] New Feature

**Checklist before requesting review:**

- [x] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [x] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

**Comment by ciaranschutte** [comment]
@hlminh2000 for the case of displaying error icon when file upload is invalid, I propose we refactor to just add `uploadError` field to `clinicalRegistrationData`. thoughts?
https://app.zenhub.com/workspaces/argo-5c8c8637777e5f767e486217/issues/icgc-argo/platform-ui/779

File: components/pages/submission-system/SIDE_MENU_SAMPLE_REGISTRATION_STATE.gql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+query SIDE_MENU_SAMPLE_REGISTRATION_STATE($programShortName: String!) {
+  clinicalRegistration(shortName: $programShortName) {
+    programShortName
+    createdAt
```
Comment (hlminh2000) [inline components/pages/submission-system/SIDE_MENU_SAMPLE_REGISTRATION_STATE.gql]: I don't think `createdAt` is used eh?

File: components/pages/submission-system/SideMenu.tsx [hunk 2]
```diff
@@ -64,6 +74,14 @@ type ClinicalSubmissionQueryResponse = {
     }>;
   };
 };
+
+type SampleRegistrationQueryResponse = {
+  clinicalRegistration: {
+    fileName: string;
```
Comment (hlminh2000) [inline components/pages/submission-system/SideMenu.tsx]: should we include `programShortName` here too to reflect the actual gql query? It's not actually read anywhere so I see why we might choose to leave it out, just curious if we have a preference

File: components/pages/submission-system/SideMenu.tsx [hunk 3]
```diff
@@ -100,7 +131,24 @@ const LinksToProgram = (props: { program: SideMenuProgram; isCurrentlyViewed: bo
...
+          content={
+            <div
+              css={css`
+                display: flex;
+                justify-content: space-between;
+                width: 100%;
+                align-items: center;
+                padding-right: 15px;
+              `}
+            >
```
Comment (hlminh2000) [inline components/pages/submission-system/SideMenu.tsx]: shouldn't this div be the `StatusMenuItem`? :)

File: components/pages/submission-system/SideMenu.tsx (hunk 2 above)
Comment (ciaranschutte) [reply components/pages/submission-system/SideMenu.tsx]: yeah i see no harm in that. minimises potential confusion

File: components/pages/submission-system/SideMenu.tsx (hunk 3 above)
Comment (ciaranschutte) [reply components/pages/submission-system/SideMenu.tsx]: nice catch! :)

**Pull request merged**